crates/accelerate/src/nlayout.rs

```diff
@@ -123,12 +115,11 @@ impl NLayout {
     }
 
     fn __reduce__(&self, py: Python) -> PyResult<Py<PyAny>> {
-        Ok((
-            py.get_type_bound::<Self>()
-                .getattr("from_virtual_to_physical")?,
-            (self.virt_to_phys.to_object(py),),
+        (
+            py.get_type::<Self>().getattr("from_virtual_to_physical")?,
+            (self.virt_to_phys.clone().into_pyobject(py).unwrap(),),
```

Comment:
```suggestion
            (self.virt_to_phys.clone().into_pyobject(py)?,),
```
since we're already in a `PyResult`-returning function.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/13577/commits/0577f48a7814455551ec2cda3540a9d1a7ad9847